include/zephyr/net/socket.h

```diff
@@ -1273,7 +1273,7 @@ struct in6_pktinfo {
  * @name Backlog size for listen()
  * @{
  */
-/** listen: The maximum backlog queue length (ignored, for compatibility) */
```

Comment:
Shall this be removed? We don't implement this option

Reply:
It is actually not an option, but just a define telling the maximum backlog length. I think we can leave it here, but we could/should probably adjust the value a bit. It seems to be quite long.